.circleci/workflows.yml

```diff
@@ -1180,6 +1181,11 @@ jobs:
             export MVN_PRJ_VERSION_PATCH=$(cat /tmp/gio.maven.project.release.version | awk -F '.' '{print $3}' | awk -F '-' '{print $1}')
             export MVN_PRJ_VERSION_QUALIFIER=$(cat /tmp/gio.maven.project.release.version | awk -F '-' '{print $2}' | awk -F 'alpha.' '{print $2}')
             
+            if [ "${MVN_PRJ_VERSION_MAJOR}.${MVN_PRJ_VERSION_MINOR}.${MVN_PRJ_VERSION_PATCH}" != "${GIO_VERSION}" ]; then
```

Comment:
**issue**: with this change we will not be able to release alpha version as alpha version includes qualifer.

you should do this check at line 1195 just before the mvn command by using the MVN_PRJ_VERSION